docs/conf/portland/2024/news/welcome.rst

```diff
@@ -0,0 +1,70 @@
+:template: {{year}}/generic.html
+
+.. post:: Oct 12, 2023
+   :tags: {{shortcode}}-{{year}}
+
+Announcing Write the Docs {{ city }} {{year}}
+===========================================
+
+Welcome to the official launch of `Write the Docs Portland {{year}} <https://www.writethedocs.org/conf/{{ shortcode }}/{{ year }}/>`_.
+We're excited to announce the dates for our next conference:
+**Write the Docs {{ city }} {{ year }} will be on {{ date.short }}**
+
+We are excited to welcome everyone back to Revolution Hall in Portland for another year of great talks and community building.
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+**We plan to launch our CFP in November** for those who are excited to speak.
+We're still working out the exact details for the CFP,
```

Comment:
Does this mean pre-record all the talks and broadcast them on the stage? I thought that we decided not to do that, but instead go back to recording the live talks and publishing them. Not sure if I missed this change or if this paragraph is leftover from a previous iteration...